lib.typ

```diff
@@ -31,7 +35,7 @@
   builtin-text: "simp",
   custom-text: none,
   offset: 0,
-  indicies: none,
+  index: none,
```

Comment:
Thank you for the correction. But I still prefer the plural form `indices`.

Reply:
OK, done in f03709d.